.github/workflows/ci.yml

```
@@ -2,167 +1,127 @@
-
-    - name: Set up Node.js
-      uses: actions/setup-node@v3
-      with:
-        node-version: '18'
-
-    - name: Publish to VS Code Marketplace
-      if: success()
-      run: |
-        npm install -g vsce
-        vsce publish -p ${{ secrets.VSCE_PAT }} --packagePath artifacts/vsix/*.vsix
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)